packages/blade/src/utils/useBreakpoint/useBreakpoint.web.test.ts

```diff
@@ -4,15 +4,15 @@ import { useBreakpoint } from './';
 import { breakpoints } from '~tokens/global';
 
 describe('useBreakpoint', () => {
-  it('should detect mobile and return smallest breakpoint of mobile devices', () => {
+  it('should return default breakpoint and device', () => {
     setupMatchMediaMock({
       matches: true,
     });
 
     const { result } = renderHook(() => useBreakpoint({ breakpoints }));
 
-    expect(result.current.matchedBreakpoint).toBe('base');
-    expect(result.current.matchedDeviceType).toBe('mobile');
```

Comment:
It should continue to detect mobile and base no? just not detect it on the initial client render 🤔